File: majority-element.py
class Solution(object):
    def majorityElement(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        k=0
        for i in (set(nums)):
            if k < nums.count(i):
                u=i
                k=max(k,nums.count(i))
        return u

File: majority-element.java
class Solution {
    public int majorityElement(int[] nums) {
        for (int i=0; i < nums.length; i++){
            int key = nums[i];
            int j = i-1;
            while( j >= 0 && nums[j] > key){
                nums[j + 1] = nums[j];
                j = j-1;
            }nums[j+1] = key;
        }

        return nums[nums.length / 2];
    }
}